package.json

```diff
@@ -70,18 +72,25 @@
     "@polymer/paper-button": "^3.0.1",
     "@polymer/paper-radio-group": "^3.0.1",
     "@polymer/paper-slider": "^3.0.1",
+    "@types/pngjs": "^3.3.2",
+    "@types/puppeteer": "^1.11.1",
     "@types/three": "^0.93.12",
     "@webcomponents/webcomponentsjs": "^2.1.3",
     "chai": "^4.1.2",
     "chokidar-cli": "^1.2.1",
     "concurrently": "^4.0.1",
     "cross-os": "^1.3.0",
+    "esm": "^3.0.84",
     "fullscreen-polyfill": "^1.0.2",
     "intersection-observer": "^0.5.1",
     "local-web-server": "^2.6.0",
+    "make-dir": "^1.3.0",
     "mocha": "^5.2.0",
+    "pngjs": "^3.3.3",
     "polymer-build": "^3.1.1",
+    "puppeteer": "^1.11.0",
     "resize-observer-polyfill": "^1.5.0",
+    "rimraf": "^2.6.2",
     "rollup": "^0.66.0",
     "rollup-plugin-cleanup": "^3.0.0-beta.1",
```

Comment:
With these new dependencies, I can definitely see the image comparison being a separate (in repo?) module in the future

Reply:
Yeah, probably. But, I think it should probably bake a bit before we consider splitting it out into its own thing.

Reply:
:100: